source/views/tasks_view.py

```diff
@@ -4,22 +4,36 @@
 from schemas import *
 # from sqlalchemy_pagination import paginate
 
+from utils import *
+
+from utils import *
+
+
 
 class TasksView(FlaskView):
     task_schema = TaskSchema()
 
+    class TasksPaginationSchema(PaginationSchema):
+        items = fields.List(fields.Nested(TaskSchema))
+
+    pagination_schema = TasksPaginationSchema()
+
     def index(self):
-        tasks = Task.query.all()
-        # page = Task.query.order_by(Task.paginate(1,10,False))
-        tasks_data = self.task_schema.dump(tasks, many=True).data
-        return jsonify({'task': tasks_data}), 200
+        user = authorization(request.headers.get('Authorization', None))
+        page = request.args.get('Page', None)
```

Comment:
args should be lower case, it's a guideline only but we are looking to set all args in lower case:
`page` and  `per_page`